src/lsm/README.md

```diff
@@ -223,7 +223,7 @@ At the end of the last beat of the compaction bar (`23`):
 ## Manifest
 
 The manifest is a tree's index of table locations and metadata.
-(Not to be confused with the [SuperBlock Manifest](../vsr/README.md#manifest)).
+(Not to be confused with the [SuperBlock Manifest](/src/vsr/superblock_manifest.zig)).
```

Comment:
@sentientwaffle maybe this fix is worth double checking. The original link doesn't exist and nothing on the vsr/README mentioned the superblock manifest.